lib/graphql/schema.flow.js

```diff
@@ -12,11 +12,8 @@ export type Scalars = {|
   Boolean: boolean,
   Int: number,
   Float: number,
-  /** The javascript `Date` as string. Type represents date and time as the ISO Date string. */
   DateTime: any,
-  /** The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
   JSON: any,
-  /** The `JSONObject` scalar type represents JSON objects as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
```

Comment:
🔧 unwanted deletions, repeated through the pr

Reply:
@OgulcanCelik solved within this commit [feda9cf](https://github.com/kontist/js-sdk/pull/390/commits/feda9cf22d6028a0c1f7f3ecc3a6361b3b16b3c0)
;
actually these comments are removed after running the command , so i forgot to revert it manually 🙏